index.html

```diff
@@ -4,73 +4,78 @@
                             </a>
                         </li>
                         <li class="nav-item">
-                            <a class="nav-link dropdown-item text-center" href="#">Program</a>
+                            <a class="nav-link text-center" href="#">Program</a>
                         </li>
                         <li class="nav-item">
-                            <a class="nav-link dropdown-item text-center" href="#">Join</a>
+                            <a class="nav-link text-center" href="#">Join</a>
                         </li>
                         <li class="nav-item">
-                            <a class="nav-link dropdown-item text-center" href="#">Sponsor</a>
+                            <a class="nav-link text-center" href="#">Sponsor</a>
                         </li>
                         <li class="nav-item">
-                            <a class="nav-link dropdown-item text-center" href="#">News</a>
+                            <a class="nav-link text-center" href="#">News</a>
                         </li>
-                        <li class="nav-item hl">
-                            <a class="nav-link dropdown-item text-center red" href="#">CC Campaign</a>
+                        <li class="nav-item border border-danger border-3">
+                            <a class="nav-link text-center text-danger" href="#">CC Campaign</a>
                         </li>
                     </ul>
                 </div>
             </div>
         </nav>
     </section>
 
-    <section class="container bg-light p-5 m-5">
-        <p class="fs-3 text-danger pt-5">"Hello! Open world"</p>
-        <h1 class="text-danger fs-1 col-md-7">OPEN SOURCE SOFTWARE CONFERENCE 2025</h1>
-        <p class="bw p-3 col-md-7">A joyfull celebration believing in the value of openness and sharing. Creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
-        <div class="fs-2 fw-bold">2025.10.15(THU) ~ 16(FRI)</div>
-        <p class="fw-bold fs-5">National Museum of Korea, Art Center Nabi and more</p>
+    <section class="container row bg-light p-5 m-5 text-start">
+        <div class="col-md-3"></div>
+        <div class="col-md-6">
+            <p class="fs-3 text-danger pt-5">"Hello! Open world"</p>
+            <h1 class="text-danger fs-1">OPEN SOURCE SOFTWARE CONFERENCE 2025</h1>
+            <p class="bw p-3">A joyfull celebration believing in the value of openness and sharing. Creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+            <div class="fs-2 fw-bold">2025.10.15(THU) ~ 16(FRI)</div>
+            <p class="fw-bold fs-5">National Museum of Korea, Art Center Nabi and more</p>
+        </div>
+        <div class="col-md-3"></div>
     </section>
     <section class="container-fluid p-5 blackTexture">
         <h3 class="text-white text-center mb-5">Main Program</h3>
```

Comment:
[OPTIONAL] Kindly fix the main program section background color like in the design provided.

<img width="870" alt="Screenshot 2023-01-07 at 6 26 17 PM" src="https://user-images.githubusercontent.com/25887253/211158344-31edf2e6-8abe-4801-bb51-002a70f068b6.png">